src/main/java/com/github/lunatrius/schematica/client/gui/control/GuiSchematicMaterialsSlot.java

```diff
@@ -19,6 +22,8 @@ public GuiSchematicMaterialsSlot(GuiSchematicMaterials par1) {
         super(Minecraft.getMinecraft(), par1.width, par1.height, 16, par1.height - 34, 24);
         this.guiSchematicMaterials = par1;
         this.selectedIndex = -1;
+        // run scanning
+        this.updateCount();
```

Comment:
A nitpick, but I usually don't prefix methods with `this`. *They are unique in any context whereas variables might be either a field, local variable, parameter, ... in the same context.*